resume.md

```diff
@@ -1,49 +1,47 @@
-# FIRST LAST NAME
+# Kelly Ma
 
-(123) 456 - 7890 | email@gmail.com
+(510) 599 - 0540 | kellyma@berkeley.edu
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+https://kelly2135ma.wixsite.com/mysite | https://www.linkedin.com/in/kelly-ma-2a1172174/
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**Galileo Academy** | *June 2020*
+**University of College** | B.S. Degree ... *May 2024*
+* **GPA**:4.0 | 
+* **Coursework**: Introduction to Programming Data Science Foundations, Linear Algebra.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python
```

Comment:
After 61B, 61C, you should have at least one strongly typed language (Java) and at least one lower level language (C / C++)